packages/scripts/src/steps/webpack/getConfig.mts

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Microsoft. All rights reserved.
  * Licensed under the MIT license. See LICENSE file in the project.
  */
-import { configure } from '@essex/webpack-config'
+import essexWebpackConfig from '@essex/webpack-config'
```

Comment:
Moving to default exports?

Reply:
I noticed some issues with importing CJS from ESM - I think it's from TypeScript. 

But basically, named imports from CJS were causing issues and we had to destructure later.